samples/attestation_plugin/enclave/plugin3/plugin3.c

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This plugin3.c seems to be irrelevant.

it would be great if this is replaced by some sample JWT-like token validation plugin.

Reply:
Yeah. I didn't delete samples 2 and 3, because I was planning to update them.